### PR TITLE
feat: add relayer command

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Closes https://github.com/celestiaorg/orchestrator-relayer/issues/48

Depends on the other PRs.

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: cmd/qgb/relayer/cmd.go [hunk 1]
```diff
@@ -1,20 +1,176 @@
...
+			}
+
+			// wait for the dht to have some peers
+			err = dht.WaitForPeers(cmd.Context(), time.Hour, 10*time.Second, 1)
 			if err != nil {
 				return err
 			}
 
-			// TODO add implementation
+			// creating thee p2p querier
```
Comment (SweeXordious) [inline cmd/qgb/relayer/cmd.go:133]:
```suggestion
			// creating the p2p querier
```

File: cmd/qgb/relayer/cmd.go [hunk 2]
```diff
@@ -1,20 +1,176 @@
...
+						if err != nil {
+							logger.Error(err.Error())
+							time.Sleep(time.Second * 30)
+							continue
+						}
+						return
+					}
+				}
+			}()
+			wg.Wait()
```
Comment (evan-forbes) [inline cmd/qgb/relayer/cmd.go:173]: [blocking question]
what's the purpose of this loop? just to restart the relayer? is that a desired property? why is a goroutine needed here?

**Review by evan-forbes — COMMENTED** [review]
only one blocking question

File: cmd/qgb/relayer/cmd.go (hunk 2 above)
Comment (SweeXordious) [reply cmd/qgb/relayer/cmd.go:173]: definitly :+1: This is how it was in the previous implementation and didn't change it. Will make the change right away. Thanks

Comment (SweeXordious) [reply cmd/qgb/relayer/cmd.go]: [e862108](https://github.com/celestiaorg/orchestrator-relayer/pull/151/commits/e8621084c91d1e7376a95aef4c003bf9596fb4a6)

@evan-forbes Like this?

**Review by evan-forbes — APPROVED** [review]
I'm still unsure if we should be restarting the relayer. If the user wants that behaviour, then they can do so using linux

thanks for fixing so fast, otherwise LGTM :+1:

**Comment by SweeXordious** [comment]
@evan-forbes done. There is an issue with `TempDir` in the CI which happens from time to time... I'm looking into, can we merge this one even if it's not solved? I will create an issue specifically for it

**Review by evan-forbes — APPROVED** [review]

**Comment by evan-forbes** [comment]
aproov :shipit:

**Pull request merged**